crates/winmd/src/types/type_name.rs

```diff
@@ -351,6 +351,13 @@ impl TypeName {
 
         TokenStream::from_iter(generics)
     }
+
+    pub fn clean_name(&self) -> &str {
+        if self.name.starts_with("#r") {
```

Comment:
Not sure why this is here. This will never be true.

Reply:
https://github.com/microsoft/winrt-rs/blob/master/crates/winmd/src/lib.rs#L25

Reply:
But that's the token representation - this is the original string name.